### ISSUE TITLE
Profile page: Not looking good empty space in sidebar (see screenshot)

### ISSUE THREAD

**Issue by michelesbw** [sbwtech/theme_sbw#57]
![screenshot-socialbusinessworld org 2016-10-29 10-35-11](https://cloud.githubusercontent.com/assets/5203511/19828354/c3352034-9dc3-11e6-947f-f79eb5293fdc.png)

**Comment by juho-jaakkola** [comment]
I don't think there is anything that can be done to this with the two-column layout.

I'm closing as wontfix. Feel free to reopen if you come up with a technical solution that fixes it.

**Comment by michelesbw** [comment]
Couldn't we set left column's activity items at 100% width so that all that empty space would disappear?

**Comment by juho-jaakkola** [comment]
I'd rather stick to one of the default layouts. Meaning either that all content are in two-column or one-column layout. But not both mixed within the same page.

**Comment by juho-jaakkola** [comment]
Custom layout structures increase complexity with too little gain.

**Comment by michelesbw** [comment]
To me it's ok 2-columns, I don't know why then it becomes 3 columns below, maybe previous theme or any setting somewhere? Widget manager?

**Comment by juho-jaakkola** [comment]
Now that I checked, the profile page uses in fact the "widgets" layout, which has the profile part at top, and three-column widget section at bottom.

The current structure has been achieved by customizing the top part for SBW's needs. I tested moving river below the other content, but there's the same problem in the cases that use has entered minimal profile information:

![image](https://cloud.githubusercontent.com/assets/883920/20151573/43579546-a6c3-11e6-8036-d5253c74afa6.png)

I'm not sure if that is any better than the current situation.
